src/System.Devices.Gpio/System.Devices.Gpio.Samples/LiquidCrystal.cs

```diff
@@ -426,14 +426,14 @@ private void Write8bits(byte value)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private void DigitalWrite(Pin pin, int value)
+        private static void DigitalWrite(Pin pin, int value)
         {
             PinValue state = HasFlag(value, 0x01) ? PinValue.High : PinValue.Low;
```

Comment:
what is the magic number 0x01 in this context? can this be an enum or const?

Reply:
It means just to set the pin to high. I will make a local constant for it.